File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/covid/ciudadano/dominio/filtroCiudadano.ts
import { ParametroPaginacionGenerico } from "src/app/erp_module/shared/dominio/ParametroPaginacionGenerico";

export class FiltroCiudadano {
    documento: string;
    nombre: string;
    pais: string;
    departamento: string;
    provincia: string;
    distrito: string;
    direccion: string;
    estado: number;
    tipodocumento: number;

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;
}

File: Administrativo/spring-erp-servicio/net/royal/spring/covid/dominio/filtro/FiltroCiudadano.cs
using net.royal.spring.framework.core.dominio;
using System;


namespace net.royal.spring.covid.dominio.filtro
{
    public class FiltroCiudadano
    {
        public String documento { get; set; }
        public String nombre { get; set; }
        public String pais { get; set; }
        public String departamento { get; set; }
        public String provincia { get; set; }
        public String distrito { get; set; }
        public Nullable<Int32> estado { get; set; }
        public Nullable<Int32> tipodocumento { get; set; }
        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroCiudadano()
        {
            paginacion = new ParametroPaginacionGenerico();
        }
    }
}
